packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -166,75 +175,77 @@ export const UIModeView: React.FC<{}> = ({
 
   const isRunningTest = !!runningState;
 
-  return <div className='vbox ui-mode'>
-    {isDisconnected && <div className='drop-target'>
-      <div className='title'>UI Mode disconnected</div>
-      <div><a href='#' onClick={() => window.location.reload()}>Reload the page</a> to reconnect</div>
-    </div>}
-    <SplitView sidebarSize={250} minSidebarSize={125} orientation='horizontal' sidebarIsFirst={true}>
-      <div className='vbox'>
-        <div className={'vbox' + (isShowingOutput ? '' : ' hidden')}>
-          <Toolbar>
-            <div className='section-title' style={{ flex: 'none' }}>Output</div>
-            <ToolbarButton icon='circle-slash' title='Clear output' onClick={() => xtermDataSource.clear()}></ToolbarButton>
-            <div className='spacer'></div>
-            <ToolbarButton icon='close' title='Close' onClick={() => setIsShowingOutput(false)}></ToolbarButton>
-          </Toolbar>
-          <XtermWrapper source={xtermDataSource}></XtermWrapper>
+  return <AppContext.Provider value={ { isUIMode: true } }>
```

Comment:
The UX: accepting the screenshot should happen next to the screenshot diff view since that's where the decision happens.